src/coveralls.jl

```diff
@@ -135,9 +135,9 @@ module Coveralls
             end
         end
 
-        r = post(URI("https://coveralls.io/api/v1/jobs"), files =
+        r = HTTP.post("https://coveralls.io/api/v1/jobs", files =
             [FileParam(JSON.json(data),"application/json","json_file","coverage.json")])
         println("Result of submission:")
-        println(Compat.UTF8String(r.data))
+        println(String(r.data))
```

Comment:
[`HTTP.Response`](https://juliaweb.github.io/HTTP.jl/latest/index.html#HTTP.Messages.Response) apparently does not have a `.data` field. This crashes Codecov (after submission), e.g. [here](https://travis-ci.org/JuliaDocs/Documenter.jl/jobs/332152832#L1234-L1242). Should be `.body` instead?

Reply:
Thanks, good catch. See #147.